config/examples/publishedresources/published-resource-instances.yaml

```diff
@@ -0,0 +1,13 @@
+apiVersion: cache.kcp.io/v1alpha1
+kind: PublishedResource
+metadata:
+  name: instances
+spec:
+  resource: instances
+  group: machines.svm.io
+  version: v1alpha1
+  labelSelector:
+    matchLabels:
+      app.kubernetes.io/part-of: instances
+status:
+  identityHash: "dummy-hash"
```

Comment:
remind me what this identityHash is. identity of what?

Reply:
Identity of resources published by this PublishedResource. Meaning enables to access them from other workspaces via dedicated VW in the future.